rust/cbindgen.toml

```diff
@@ -109,6 +109,7 @@ exclude = [
     "IPPROTO_TCP",
     "IPPROTO_UDP",
     "SRepCatGetByShortname",
+    "SIG_FLAG_TOSERVER",
```

Comment:
@satta why is MQTT detection only to server ?

Reply:
Good question. I just grepped through `detect-mqtt-*` and saw many instances of `SIG_FLAG_TOSERVER`... well, not all of the things that are alerted on in the detection keywords are sent to the broker, but most are (e.g. anything with CONNECT or PUBLISH, but not CONNACK, the reason code, etc.).
Could be a copy-paste artifact; interesting that these keywords still seem to work though.

Maybe I should also revisit the code and check if maybe the transactions the parser produces are deliberately registered as being toserver because I didn't know an alternative (even though they likely contain messages in various directions) and that's why we want to apply the keywords to toserver.

Reply:
And there is no `SIG_FLAG_TOCLIENT`